engine/src/main/java/org/terasology/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -48,6 +47,7 @@
  * - https://docs.google.com/drawings/d/13I0GM9jDFlZv1vNrUPlQuBbaF86RPRNpVfn5q8Wj2lc/edit?usp=sharing
  */
 public class ShadowMapNode extends AbstractNode {
+    private static final String SHADOW_MAP_FBO_NAME = "sceneShadowMap";
```

Comment:
Hmmm, my thinking behind this is making distinction between a real `FBO` and a name of an `FBO`.  But it seems like you are strongly against `NAME` 😄, let me remove them! :tada: